### PR TITLE
chore: move cli to main repo

### PR DESCRIPTION
**Pull request by FrankEssenberger**
Move the https://github.com/SAP/cloud-sdk-cli to the main repo.
A package for the long running tasks is added for nightly execution.
The package is set to private so not yet published.

The huge number of files in the `template` folder is just copied and used to generate new projects. No need to review there. Also the test itself are not changed. Only some changes to the different module handling between the two projects.

### CODE REVIEW

**Review by jjtang1985 — CHANGES_REQUESTED** [review]

**Review by jjtang1985 — CHANGES_REQUESTED** [review]
I'm happy to see the execution time of the pipeline is not affected.

**Comment by jjtang1985** [comment]
I failed to make the review in one action..

File: packages/cli/package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  "keywords": [
+    "sap-cloud-sdk",
+    "cloud-sdk",
+    "sap-cloud-platform",
+    "cli"
+  ],
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "publishConfig": {
+    "access": "private"
```
[the remark this answers is not in the document]
Comment (FrankEssenberger) [reply packages/cli/package.json:17]: I have create some follow up tickets I would like to do before releasing.

**Review by jjtang1985 — APPROVED** [review]
LGTM

**Pull request merged**